fabtests/common/shared.c

```diff
@@ -3154,10 +3190,14 @@ int ft_check_buf(void *buf, size_t size)
 	if (i != size) {
 		printf("Data check error (%c!=%c) at byte %zu for "
 		       "buffer size %zu\n", c, recv_data[i], i, size);
-		return -FI_EIO;
+		ret = -FI_EIO;
+		goto out;
```

Comment:
nit: out is the next line, so the goto isn't needed.

Reply:
@shefty Updated